test/test_datapipe.py

```diff
@@ -6,13 +6,16 @@
 
 import io
 import itertools
+import os
```

Comment:
@tmbdev sorry can you revert the changes to this file too?

Reply:
thank you so much!!!

Reply:
Done. (I'm surprised the linter doesn't check for unused imports.)